### PR TITLE
Fetch Hello Message

### PR DESCRIPTION
**Pull request by connielei**
The page now contains a button that adds the "Hello Connie!" message to the DOM.

https://3-2-dot-cool-project2020.appspot.com/

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -26,3 +26,12 @@ function addRandomGreeting() {
...
 }
+
+/**
+ * Fetches the hello message and adds it to the page.
+ */
+async function getHello() {
+    const response = await fetch('/data');
+    const hello = await response.text();
+    document.getElementById('hello-container').innerText = hello;
+}
```
Comment (Enriqe) [inline portfolio/src/main/webapp/script.js]: Nit: please add a new line at the end of the file

**Comment by Enriqe** [comment]
When possible, can you link a demo / screenshot here as well please?

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -26,3 +26,12 @@ function addRandomGreeting() {
...
   greetingContainer.innerText = greeting;
 }
+
+/**
+ * Fetches the hello message and adds it to the page.
+ */
+async function getHello() {
+    const response = await fetch('/data');
+    const hello = await response.text();
+    document.getElementById('hello-container').innerText = hello;
```
Comment (newmuis) [inline portfolio/src/main/webapp/script.js]: `textContent` is preferred over `innerText`

There are a lot of details on why ([example](https://dev.to/4myc/javascript-innerhtml-innertext-and-textcontent-ih)), but tl;dr it's faster and potentially safer

Comment (connielei) [reply portfolio/src/main/webapp/script.js]: Thanks for letting me know, I didn't know textContent existed.

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -99,3 +99,12 @@ function displayRandomProject() {
   updateProjectLink(project.link);
   updateProjectImage(project.image);
 }
+
+/**
+ * Fetches the hello message and adds it to the page.
+ */
+async function getHello() {
```
Comment (Enriqe) [inline portfolio/src/main/webapp/script.js:106]: maybe call it something more generic in case the message changes? `getGreeting` maybe

Comment (connielei) [reply portfolio/src/main/webapp/script.js:106]: I change it in the next PR to getMessages and then eventually to getComments, should I change it here still?

Comment (Enriqe) [reply portfolio/src/main/webapp/script.js:106]: That's fine, it can wait till the next PR.

**Review by Enriqe — APPROVED** [review]
🛳

**Review by Enriqe — CHANGES_REQUESTED** [review]
Oops sorry, I think there's something wrong with the parsing? I'm seeing the `<h1>` tags on the message when it gets appended

**Review by Enriqe — APPROVED** [review]

**Pull request merged**